### ISSUE TITLE
[Bug]: CORS blocking download of export on shared canvas

### ISSUE THREAD

**Issue by mklilley** [tldraw/tldraw#3140]
### What happened?

Failed to export image from a shared canvas (i.e. via the editable sharable link). Dev tools tells of CORS problem.
> Access to fetch at 'https://assets.tldraw.xyz/uploads/nAl6aVe70SH3vo0jMw2Vd-ikigai.svg' from origin 'https://www.tldraw.com' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'null' that is not equal to the supplied origin. Have the server send the header with a valid value, or, if an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

### How can we reproduce the bug?

- Create a new drawing on tldraw.com
- Click the share button
- Click "Share this project"
- Copy the share link and paste into a new tab
- Go to menu -> edit -> export as PNG or SVG
- Download will fail

### What browsers are you seeing the problem on?

Chrome

### Contact Details

m.k.lilley@gmail.com

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mklilley** [comment]
Seems to have fixed itself somehow 🤷‍♂️

**Comment by cdosborn** [comment]
I'm running into this issue too, cannot export for this reason

**Comment by mangkoran** [comment]
Just now I also encountered this issue when I'm trying to export shared canvas

**Comment by mimecuvalo** [comment]
Copying response from this other bug: https://github.com/tldraw/tldraw/issues/3986

Thanks for reporting! We have recently changed how our CORS fetching works so it seems like this started hitting a particular configuration with Cloudflare and caching.

I modified the Cloudflare Browser Cache TTL to not override and set it to 4 hours and instead to "Respect Existing Headers" since we have a worker specifically to handle this.

This is fixed now.
